tools/licenses/lib/licenses.dart

```diff
@@ -752,6 +752,13 @@ Iterable<_LicenseMatch> _tryReferenceByFilename(String body, LicenseFileReferenc
       if (match.getCopyrights() == '') {
         yield new _LicenseMatch(template, match.start, match.end, debug: '_tryReferenceByFilename (with failed copyright search) looking for $filename');
       } else {
+        License notice = new License.unique(body.substring(match.start, match.end), LicenseType.unknown, origin: origin);
+        yield new _LicenseMatch(
+            new License.unique(body.substring(match.start, match.end), LicenseType.unknown, origin: origin),
```

Comment:
i assume you meant to use `notice` here rather than duplicating this line. Or just remove the line creating `notice` above.